<!-- Mission de suivi de waypoints -->
<!-- Morse + bateau -->
<launch>
	<arg name="show_plot" default="False"/>
	<node name="cmd_ramped" pkg="actuators" type="cmd_to_ramp.py">
		<param name="linear_twist" value="0"/>
		<param name="angular_twist" value="0"/>
		<param name="angular_accel" value="10"/> 
		<param name="linear_accel" value="10"/>
		<remap from="cmd_vel_ramped" to="boat/actuator"/>
	</node>

	<node pkg="rviz_satellite" type="map2world_tf_broadcaster.py"
        name="broadcaster_fixed" />

	<node name="regulateur_champs" pkg="process" type="regulateur_champs_boat2_morse.py" output="screen">
		<param name="V0" value="0"/>
		<param name="vHigh" value="10"/>
		<param name="vLow" value="0.5"/>
		<param name="Kp" value="30"/>
		<param name="Kd" value="2"/>
		<param name="V_lim_reverse" value="1.5"/>
		<param name="reverse_motor" value="1"/> 	<!-- (-1) to reverse -->
		<param name="robot_type" value="thomas_boat"/>
		<remap from="gps/local_pose" to="boat/gps/pose"/>
	</node>

	<node name="behavior_server" pkg="process" type="behavior_server.py">
		<param name="robot_type" value="thomas_boat"/>
		<remap from="gps/local_pose" to="boat/gps/pose"/>
		<param name="show_plot" value="$(arg show_plot)"/>
	</node>

	<node name="gui_behavior_adder" pkg="process" type="behavior_adder_gui.py"/>
</launch>
